---
bibliography: CLER.bib
---

<!-- Person responsible: de Flamingh -->

# Landscape genetics {#sec-genetics}

Genetic data can be applied in complement with ecological data to integrate evolutionary processes and patterns into landscape ecology [@Holderegger2008LandscapeGenetics; @Manel2003LandscapeGenetics; @Balkenhol2017LandscapeGenomics].
Landscape genetics unites molecular population genetics, spatial statistics, and landscape ecology, and emerged from the goal to study the interaction between landscape features and microevolutionary processes, such as gene flow, genetic drift and selection [@Manel2003LandscapeGenetics; @Balkenhol2017LandscapeGenomics].
Landscape genetic approaches have traditionally been used to inform landscape ecology applications, e.g., through connectivity modelling and spatial conservation planning [@Manel2003LandscapeGenetics; @Holderegger2008LandscapeGenetics; @Manel2013TenYears; @Bolliger2014LandscapeGenetics].
Using holistic approaches that consider evolutionary processes and patterns in addition to ecological data can fortify results, e.g., by improving predictive models of species range shifts in response to climate change [e.g., @Aguirre-Liguori2021EvolutionaryGenomics], improving the identification and delineation of landscape connections among populations [e.g., @Homola2019LandscapeGenetics; @deFlamingh2024IntegratingHabitat], allowing for the interpretation of spatial structuring in context of socio-cultural connections [e.g., @Henson2021ConvergentGeographic], or understanding local adaptation associated with specific environments [@Sork2013PuttingLandscape; @Rolland2015CombiningNiche; @Aguirre-Liguori2021EvolutionaryGenomics].

Advances in molecular methodologies and synergistic developments of bioinformatic and computational approaches to analyze large-scale genomic data have recently enabled the integration of genome-wide data with spatial ecological data.<!-- MH: Ref missing -->.
This shift towards landscape genomics [@Cushman2018EditorialLeast; @Storfer2018NavigatingInterface] comes with novel utilities through which researchers can apply genomic data across spatio-temporal scales, including the integration of historic or ancient DNA [@Fenderson2020SpatiotemporalLandscape], or simulated present-day and future population genomic data [@House2018EvaluatingMethods; @Chevy2024PopulationGenetics].
Analytical frameworks such as the *FOLDS* model [gene flow, genetic offsets, genetic load, dispersal and SDMs\; @Aguirre-Liguori2021EvolutionaryGenomics], constrained coordination where covarying sets of genotypes are correlated with multivariate environments through redundancy analysis [RDA\; @Capblancq2021RedundancyAnalysis], and spatial simulations provide promising potential for future landscape genomic studies.
Such studies may explore interactions between complex evolutionary processes, including demography, multidimensional (e.g., strength and direction) gene flow and migration across spatio-temporal scales, genetic differentiation across landscapes, genomic load and adaptive potential, and the interpretation of these evolutionary processes in context of landscape ecology [@Balkenhol2017LandscapeGenomics; @House2018EvaluatingMethods; @Fenderson2020SpatiotemporalLandscape; @Marcus2021FastFlexible; @Chevy2024PopulationGenetics].

Given the importance of connectivity, all previously discussed connectivity software are also relevant for landscape genetics.
Additionally, popular software for landscape genetics includes the R packages *graph4lg* [@Savary2021Graph4lgPackage], *landgenreport* [@Gruber2015LandgenreportNew], *adegenet* [@Jombart2008AdegenetPackage], or *ape* [@Paradis2019Ape50], among others.
Furthermore, a series of Python scripts have been developed for use in landscape genetics studies [@Etherington2011PythonBased], or GIS-based toolboxes written in Python exist that provide landscape genetics functionality [e.g., @Brown2017SDMtoolbox20].
